plugins/speculation-rules/settings.php

```diff
@@ -109,7 +122,7 @@ function plsr_register_setting(): void {
 		array(
 			'type'              => 'object',
 			'description'       => __( 'Configuration for the Speculation Rules API.', 'speculation-rules' ),
-			'sanitize_callback' => 'plsr_sanitize_setting',
+			'sanitize_callback' => 'plsr_sanitize_setting', // TODO: Is this even needed here due to the schema?
```

Comment:
I guess it is needed because the JSON schema is only used in a REST API request. But the `plsr_sanitize_setting()` function can reuse the schema.